app/views/TripPointDetailsView.tsx

```diff
@@ -1,296 +1,452 @@
+                  : ADD_NOTIFICATION_ICON_MATERIAL,
+                color: theme.colors.onSurface,
+                onPress: async () => {
+                  if (notificationId && tripPoint) {
+                    try {
+                      await cancelNotification(notificationId);
+                      await unregisterNotification(tripPoint.id);
+                    } catch (error: any) {
+                      console.error("Error while setting notification", error);
+                      showSnackbar("Wystąpił błąd", "error");
+                    }
+                  } else {
+                    console.log(`Dodaj`);
+                    setIsNotificationFormVisible(true);
+                  }
+                },
+              },
+            ),
+            {
+              title: "Usuń",
+              icon: DELETE_ICON,
+              color: theme.colors.error,
+              onPress: () => {
+                showRemovalModal();
+              },
+            },
+          ],
+        },
+      ],
+    });
+  }, [tripDay, navigation, notificationId, tripPoint, tripDay]);
+
+  const handleScheduleNotification = async (
+    minutes: number,
+  ): Promise<boolean> => {
+    if (!tripPoint) return false;
+
+    if (!tripDay?.date) {
+      showSnackbar("Wystąpił błąd", "error");
+      return false;
+    }
+
+    const parsedDate = new Date(
+      new Date(tripDay?.date).getTime() - minutes * 60 * 1000,
+    );
+
+    if (parsedDate < new Date()) {
+      showSnackbar("Nie można ustawić przypomnienia w przeszłości", "error");
+      return false;
+    }
+
+    const newNotificationId = await schedulePushNotification(
+      tripPoint.name,
+      `Zaczyna się za ${formatMinutes(minutes)}`,
+      parsedDate,
+    );
+    await registerNotification(tripPoint.id, newNotificationId);
+
+    return true;
+  };
+
+  if (loading) return <LoadingView />;
+
+  if (error) {
+    router.back();
+    showSnackbar(error?.toString() || "Unknown error", "error");
+    return null;
+  }
+
+  return (
+    <>
+      <GestureHandlerRootView>
+        <ScrollView style={styles.container}>
+          {Object.entries(parsedTripPoint)
+            .filter(([key]) => key in LABELS)
+            .map(([key, value]) => (
+              <TripPointDetailsLabel
+                key={key}
+                title={LABELS[key]}
+                element={
+                  <Text style={styles.value}>
+                    {value?.toString() || "Brak danych"}
+                  </Text>
+                }
+              />
+            ))}
+          {
+            <TripPointDetailsLabel
+              title="Powiązana atrakcja"
+              element={
+                tripPoint?.place?.providerId ? (
+                  <SimplePlaceCard place={convertPlace(tripPoint?.place)} />
+                ) : (
+                  <Text style={styles.value}>Brak danych</Text>
+                )
+              }
+            />
+          }
```

Comment:
I think something went wrong there because you have overriden my changes that added opening hours